google-cloud-core/src/test/java/com/google/cloud/StringEnumTest.java

```diff
@@ -110,8 +117,12 @@ public void testEquals() {
 
   @Test
   public void testValueOfStrict_invalid() {
-    expectedException.expect(IllegalArgumentException.class);
-    Letter.valueOfStrict("NonExistentLetter");
+    try {
+      Letter.valueOfStrict("NonExistentLetter");
+      Assert.fail();
+    } catch (IllegalArgumentException ex) {
+      assertNotNull(ex.getMessage());
+    }
   }
```

Comment:
Here and below in tests for IllegalArgumentException it's better to 
1) Assert.fail("Expected an IllegalArgumentException ")
2) assertThat(ex.getMessage(), is("..."));